charts/adot-exporter-for-eks-on-ec2/templates/aws-for-fluent-bit/configmap.yaml

```diff
@@ -88,6 +88,8 @@ data:
         log_group_name      /aws/containerinsights/{{ .Values.clusterName }}/application
         log_stream_prefix   ${HOST_NAME}-
         auto_create_group   true
+        {{ if .Values.fluentbit.output.applicationLog.log_retention.enabled }}log_retention_days  {{ .Values.fluentbit.output.applicationLog.log_retention.days }}{{ end }}  
+        log_retention_days  {{ .Values.fluentbit.kubernetesFilter.match }}
```

Comment:
Is this supposed to be here?  What does it do?

Reply:
@martinwangjian , could you please respond to @Aneurysm9 question? I think, this line is not required.. 

log_retention_days  {{ .Values.fluentbit.kubernetesFilter.match }}

Reply:
my bad, it's not supposed to be here, fixed